src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -633,45 +638,45 @@ static void window_cheats_money_mousedown(rct_window* w, rct_widgetindex widgetI
             widget_invalidate_by_class(WC_CHEATS, WIDX_MONEY_SPINNER);
             break;
         case WIDX_ADD_MONEY:
-            game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_ADDMONEY, _moneySpinnerValue, GAME_COMMAND_CHEAT, 0, 0);
+            CheatsSet(CheatType::AddMoney, _moneySpinnerValue);
             break;
         case WIDX_YEAR_UP:
-            year_spinner_value++;
-            year_spinner_value = std::clamp(year_spinner_value, 1, MAX_YEAR);
+            _yearSpinnerValue++;
+            _yearSpinnerValue = std::clamp(_yearSpinnerValue, 1, 8192);
```

Comment:
Should not have been replaced with a naked number. 8192 is not arbitrary, it's the maximum year that OpenRCT2 can handle currently.

Reply:
Probably slipped thru the rebase conflicts.